apps/system/test/marionette/text_selection_test.js

```diff
@@ -241,7 +241,22 @@ marionette('Text selection >', function() {
           });
         });
 
-      test('bug1119126 : Shortcut bubble should hide when system is resized',
+      // Enable the test til bug 1120750 is merged.
+      test('bug1120750 : Send out carets position for the short cut mode ',
+        function() {
+          fakeTextselectionApp.longPress('BugCenterInput');
+          var originalLocation = fakeTextselectionApp.textSelection.location;
+          fakeTextselectionApp.cut('BugCenterInput');
+
+          fakeTextselectionApp.longPress('BugBottomInput');
```

Comment:
Use tap for appearing shortcut is enough. Use long press maybe wait too long time so the shortcut has been timeout.